release-notes.md

```diff
@@ -366,6 +373,64 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v46.0
+
+This release has been tracked in [v46.0
+group](https://github.com/orgs/cloud-hypervisor/projects/6/views/4?filterQuery=release%3A%22Release+46%22)
+of our [roadmap project](https://github.com/orgs/cloud-hypervisor/projects/6/).
+
```

Comment:
I think we should include something about the improved error reporting from @phip1611

Reply:
Until #7066, the changes are not really user-visible. I'm not sure about your roadmap but I guess the new error messages on failure will land in v47 🤔 (if you want to release v46 ASAP)

Reply:
Yes. That's exactly what I am planning and I added #7066 to next release here: https://github.com/orgs/cloud-hypervisor/projects/6/views/7.

Thanks for reminding me of the disk image locking. I guess I forgot to sort the merged PR based on date last modified and only reviewed PRs opened since last release. Adding it to the release note shortly.